baybe/recommenders/pure/nonpredictive/sampling.py

```diff
@@ -83,30 +86,88 @@ class FPSRecommender(NonPredictiveRecommender):
     )
     """See :func:`baybe.utils.sampling_algorithms.farthest_point_sampling`."""
 
+    grid_density: int = field(default=10, kw_only=True)
+    """Number of points to sample per dimension for continuous subspaces."""
+
     @override
-    def _recommend_discrete(
+    def _recommend_hybrid(
         self,
-        subspace_discrete: SubspaceDiscrete,
+        searchspace: SearchSpace,
         candidates_exp: pd.DataFrame,
         batch_size: int,
-    ) -> pd.Index:
-        # Fit scaler on entire search space
-        from sklearn.preprocessing import StandardScaler
-
-        # TODO [Scaling]: scaling should be handled by search space object
+    ) -> pd.DataFrame:
+        # If space is purely discrete, reuse _recommend_discrete logic
+        if searchspace.type == SearchSpaceType.DISCRETE:
```

Comment:
Do we need this here? Isnt the discrete dealt with in the other function? (see https://github.com/emdgroup/baybe/blob/8a2c305810289883841d9284efdb25cae36ee825/baybe/recommenders/pure/base.py#L279)
Also I think this is wrong as now fpsample discrete recommendation recomends at random rather than as it should - see old `_recommend_discrete` method
Also the old `_recommend_discrete` should not be deleted I think